ibis/backends/base/sql/alchemy/client.py

```diff
@@ -90,14 +88,9 @@ def to_shapely(row, name):
     return df
 
 
-class AlchemyDialect(Dialect):
-
-    translator = AlchemyExprTranslator
-
-
 class AlchemyClient(SQLClient):
 
-    dialect = AlchemyDialect
+    _compiler = AlchemyCompiler
```

Comment:
Do we need to keep this protected?

Reply:
agree, there is no harm in having this public and its just more readable to subclasses